sources/packages/backend/libs/test-utils/src/factories/student.ts

```diff
@@ -23,6 +23,7 @@ export function createFakeStudent(user?: User): Student {
     phone: faker.phone.phoneNumber(),
   };
   student.sinConsent = true;
+  student.disabilityStatus = DisabilityStatus.NotRequested;
```

Comment:
addressing https://github.com/bcgov/SIMS/pull/2115/files#r1265791873 can eliminate this